### PR TITLE
Fix issues in analytics page

### PR DESCRIPTION
**Pull request by totop716**

### CODE REVIEW

File: src/components/AnalyticsSearch/index.tsx [hunk 1]
```diff
@@ -0,0 +1,361 @@
...
+        <Box display='flex'>
+          <SearchIcon />
+        </Box>
+      </Box>
+      {menuOpen && (
+        <div ref={wrapperRef} className='searchWidgetContent'>
+          <h6>{t('pairs')}</h6>
+          {filteredPairs.slice(0, pairsShown).map((val, ind) => {
+            const currency0 = new Token(
+              ChainId.MATIC,
```
Comment (RitzyDevBox) [inline src/components/AnalyticsSearch/index.tsx:297]: store the chain id in a const and add a comment with a todo to support multichain

Comment (totop716) [reply src/components/AnalyticsSearch/index.tsx]: Fixed the issue and pushed a commit

File: src/components/AnalyticsSearch/index.tsx [hunk 2]
```diff
@@ -0,0 +1,367 @@
...
+                return (
+                  pair[field].symbol.match(
+                    new RegExp(escapeRegExp(searchVal), 'i'),
+                  ) ||
+                  pair[field].name.match(
+                    new RegExp(escapeRegExp(searchVal), 'i'),
+                  )
+                );
+              }
+              if (field === 'token1') {
```
Comment (FriskyHamTitz) [inline src/components/AnalyticsSearch/index.tsx:150]: What's the different between this and the if above, why not combine them into 1?

**Review by FriskyHamTitz — APPROVED** [review]

File: src/components/AnalyticsSearch/index.tsx (hunk 2 above)
Comment (totop716) [reply src/components/AnalyticsSearch/index.tsx]: Fixed this and pushed another commit

**Pull request merged**